modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/table-view/DropZone.es.js

```diff
@@ -100,17 +101,18 @@ const DropZone = ({fields, onAddFieldName, onRemoveFieldName}) => {
 				columns={fields.map(({label, name}) => ({
 					key: label,
 					value: (
-						<div className="container p-0">
-							<div className="align-items-center row">
-								<div className="autofit-col-expand col">
+						<ClayLayout.ContainerFluid className="p-0">
+							<ClayLayout.ContentRow verticalAlign="center">
+								<ClayLayout.ContentCol expand>
 									{label ? label.en_US : ''}
-								</div>
+								</ClayLayout.ContentCol>
+
 								{Object.prototype.hasOwnProperty.call(
 									appliedFilters,
 									name
 								) && (
 									<ClayTooltipProvider>
-										<div className="col text-right">
+										<ClayLayout.ContentCol>
```

Comment:
what about the `text-right`?

Reply:
If the previous `autofit-col` is an `expand`, this one will be pushed to the right side

Reply:
yeah you are right in the end we have only an icon inside it